src/js/components/utils/colors.js

```diff
@@ -26,4 +39,12 @@ export const colorIsDark = (color) => {
   return (brightness < 125);
 };
 
-export default { colorForName, colorIsDark };
+export function getRGBColor(color, opacity) {
```

Comment:
I think this should be renamed `getRGBA`, since it includes alpha. And, I think `RGBA` is distinctive enough to not need the `Color` suffix.

Reply:
agreed. thanks for the suggestion.